DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc

```diff
@@ -209,13 +211,20 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
        histo[SiPixelPhase1TrackClustersOnTrackShape].fill(shape?1:0,id, &iEvent);
       }
 
+      for (int i =0; i<cluster.size(); i++){
+          SiPixelCluster::Pixel vecipxl = cluster.pixel(i);
```

Comment:
hi @akhatiwa - can this be a const & SiPixelCluster::Pixel instead so to avoid the creation of a copy here?

Reply:
@davidlange6 - that's probably better.